survey-runner-secrets/secrets/sdc-submission-encryption-sdx-private-key.pem

```diff
@@ -0,0 +1,30 @@
+-----BEGIN ENCRYPTED PRIVATE KEY-----
```

Comment:
Put into a separate 'test-keys' or similar folder that had to be manually copied here